prod-aws/kafka-shared-msk/payment-platform/notifications-raw.tf

```diff
@@ -0,0 +1,58 @@
+resource "kafka_topic" "notifications_worldpay_raw" {
+  name               = "payment-platform.notifications.worldpay.raw"
+  replication_factor = 3
+  partitions         = 15
+  config = {
+    "compression.type" = "zstd"
+    "retention.bytes"  = "3758096384"
```

Comment:
Is this intentional set to 3.5 GB? There is already a retention in time days.

Reply:
Will remove the byte retention then :saluting_face: